n2y/database.py

```diff
@@ -81,8 +81,15 @@ def _init_icon(self, icon_notion_data):
     def parent(self):
         if self.notion_parent["type"] == "workspace":
             return None
-        else:
-            return self.client.get_page(self.notion_parent["page_id"])
+
+        if "page_id" not in self.notion_parent:
```

Comment:
When would this happen? E.g., is it because the parent is actually a database? I think the only valid parent types are workspace, database, and page. Instead of returning none here, perhaps there's a better way to handle the case?

Reply:
Ah, that makes sense. The new commit returns the database by using the `block_id`.